lib/hanami/cli/generators/gem/app.rb

```diff
@@ -54,7 +55,13 @@ def generate_app(app, context) # rubocop:disable Metrics/AbcSize
             fs.write("app/action.rb", t("action.erb", context))
             fs.write("app/view.rb", t("view.erb", context))
             fs.write("app/views/helpers.rb", t("helpers.erb", context))
-            fs.write("app/templates/layouts/app.html.erb", File.read(File.join(__dir__, "app", "layouts_app.html.erb")))
+            fs.write("app/templates/layouts/app.html.erb", t("app_layout.erb", context))
+
+            if context.generate_assets?
+              fs.write("app/assets/javascripts/app.js", t("app_js.erb", context))
+              fs.write("app/assets/stylesheets/app.css", t("app_css.erb", context))
```

Comment:
What do you think about using `/js/` and `/css/` dirs instead of these rather long names? That would fit well with our short `js` and `css` asset helpers.

(This is not critical, but I figure now is as good a time as any to consider these names)

Reply:
@timriley Good one that could work.

To your knowledge, this requires a change in `hanami/hanami-js` (the Esbuild plugin), which is doable.
Not complaining about work, just sharing awareness. 😜

Reply:
@timriley Change in this PR: https://github.com/hanami/cli/pull/99/commits/264782eac857460a0e6d4b4b0c96353fae60ac5c

Reply:
@timriley Done.